tests/sles4sap/publiccloud/qesap_terraform.pm

```diff
@@ -58,29 +58,35 @@ sub set_var_output {
 sub create_playbook_section {
     # Cluster related setup
     my @playbook_list;
-    my @hana_playbook_list = (
-        "pre-cluster.yaml", "sap-hana-preconfigure.yaml -e use_sapconf=" . set_var_output("USE_SAPCONF", "true"),
-        "cluster_sbd_prep.yaml", "sap-hana-storage.yaml",
-        "sap-hana-download-media.yaml", "sap-hana-install.yaml",
-        "sap-hana-system-replication.yaml", "sap-hana-system-replication-hooks.yaml",
-        "sap-hana-cluster.yaml"
-    );
-
-    # Add registration module - "QESAP_SCC_NO_REGISTER" skips scc registration via ansible
+    my @hana_playbook_list;
+
+    # Add registration module as first element- "QESAP_SCC_NO_REGISTER" skips scc registration via ansible
     unless (get_var("QESAP_SCC_NO_REGISTER")) {
         my $reg_playbook = "registration.yaml -e reg_code=" . get_required_var("SCC_REGCODE_SLES4SAP") . " -e email_address=''";
         push(@playbook_list, $reg_playbook);
     }
 
-    if ($ha_enabled == 1) {
+    # SLES4SAP/HA related playbooks
+    if ($ha_enabled) {
+        push(@hana_playbook_list, "pre-cluster.yaml");
+        push(@hana_playbook_list, "sap-hana-preconfigure.yaml -e use_sapconf=" . set_var_output("USE_SAPCONF", "true"));
+        if (get_var("FENCING_MECHANISM") eq "sbd") { push(@hana_playbook_list, "cluster_sbd_prep.yaml"); }
```

Comment:
push(@hana_playbook_list, "cluster_sbd_prep.yaml")     if (get_var("FENCING_MECHANISM") eq "sbd") ;

Reply:
+1 to Michele's suggestion, but also using `check_var()`:
```
push(@hana_playbook_list, "cluster_sbd_prep.yaml") if (check_var('FENCING_MECHANISM', 'sbd') );
```

Reply:
Oh, such an ellegant line. I want to put it into gallery. 
Thanks both :)
Code updated.